velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +91,24 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseWeibullCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double a, double b, double p) {
+    static constexpr double kInf = std::numeric_limits<double>::infinity();
+
+    VELOX_USER_CHECK((p >= 0) && (p <= 1), "p must be in the interval [0, 1]");
+    VELOX_USER_CHECK_GT(a, 0, "a must be greater than 0");
+    VELOX_USER_CHECK_GT(b, 0, "b must be greater than 0");
+
+    if ((a == kInf) || (b == kInf)) {
+      result = 0.0;
+    } else {
+      result = b * std::pow(-std::log1p(-p), 1.0 / a);
```

Comment:
Please add a comment linking to the java code here.

Reply:
Added the comment to the function in the Java file.